actionview/test/abstract_unit.rb

```diff
@@ -101,6 +101,8 @@ def config
 
 class ActionDispatch::IntegrationTest < ActiveSupport::TestCase
   def self.build_app(routes = nil)
+    ActionDispatch::Request::Flash.use!
```

Comment:
Isn't this going to prepend the module more than once?

Reply:
Should be fine:

```
irb(main):001:0> module T; end
irb(main):002:0> 10.times { NilClass.prepend(t) }
irb(main):004:0> NilClass.ancestors

=> [T, NilClass, Object, PP::ObjectMixin, Kernel, BasicObject]
```